Homework2/task8/src/homework/Product.java

```diff
@@ -0,0 +1,73 @@
+package homework;
+
+import java.math.BigDecimal;
+
+/**
+ * Class Product has structure of products. It describes all attributes of products
+ */
+public class Product {
+
+    /**
+     * The type of the product
+     */
+    private String type;
+
+    /**
+     * The name of the product
+     */
+    private String name;
+
+    /**
+     * The count of the products
+     */
+    private int count = 0;
```

Comment:
Please be consistent - if you setup default values to any fields, setup it to every filed